deb/deb_test.go

```diff
@@ -523,7 +523,7 @@ func TestMultilineFields(t *testing.T) {
 		Info: nfpm.WithDefaults(&nfpm.Info{
 			Name:        "multiline",
 			Arch:        "riscv64",
-			Description: "This field is a\nmultiline field\nthat should work.",
+			Description: "This field is a\nmultiline field\n\nthat should work.",
```

Comment:
shouldn't it be?
```golang
			Description: "This field is a\nmultiline field\n.\nthat should work.",
```

Reply:
This PR adds the dot when creating the control file. After all, you wouldn't want the dot for RPM etc.

Reply:
Ah right, I'm blind